### DISCUSSION TITLE
View collection select query

### DISCUSSION THREAD

**Discussion by Oggy4th** [pocketbase/pocketbase#2335] [Q&A] [answered]
Can I use this kind of select query in view collection? 

```
SELECT (ROW_NUMBER() OVER()) as id, json_group_array(json_object('username',username)) FROM users
```

I wanna store some data inside view collection column.

**Comment by ganigeorgiev** [comment, marked as answer]
You can use any valid SQLite statement as long as the column has a valid field name, aka. you need to add named alias for the second select column.

But I don't understand the example and what is the desired outcome.

**Reply by Oggy4th** [reply]
@ganigeorgiev 

When every time user sends an HTTP request, my HTTP function collects a lot of data from different collections.

I thought the view collection might help. Instead of calling an HTTP function, just store those data inside the view collection by JSON or array. Then the user can get data easily using `getOne()`. No HTTP function is needed.
